src/storage/manager.rs

```diff
@@ -20,8 +20,18 @@ pub(crate) enum ManagerStorage {
     #[cfg(feature = "stronghold")]
     /// Stronghold storage.
     Stronghold,
+    #[cfg(feature = "rocksdb")]
```

Comment:
I know it was like that for stronghold already but can you put the attribute after the doc comment please ? That's what we mostly do.